src/senaite/ast/setuphandlers.py

```diff
@@ -254,6 +292,148 @@ def search_by_title(container, title):
     return filter(lambda obj: api.get_title(obj) == title, objs)
 
 
+def setup_workflows(portal):
+    """Setup workflow changes (status, transitions, permissions, etc.)
+    """
+    logger.info("Setup workflows ...")
+    for wf_id, settings in WORKFLOWS_TO_UPDATE.items():
+        update_workflow(portal, wf_id, settings)
```

Comment:
maybe better call the function like this:
`update_workflow(portal, wf_id, **settings)`

and change the signature as follows:

```python
update_workflow(portal, workflow_id, states=None, transitions=None):
    if states is None:
        states = {}
    if transitions is None:
        transitions = {}
```
This would be a bit more descriptive and the same for the other functions.